worlds/tunic/options.py

```diff
@@ -135,14 +136,20 @@ class LaurelsLocation(Choice):
     default = 0
 
 
+
 class ShuffleLadders(Toggle):
     """Turns several ladders in the game into items that must be found before they can be climbed on.
     Adds more layers of progression to the game by blocking access to many areas early on.
     "Ladders were a mistake." —Andrew Shouldice"""
     internal_name = "shuffle_ladders"
     display_name = "Shuffle Ladders"
+    
+    
+class TUNICPlandoConnections(PlandoConnections):
+    entrances = {portal.name for portal in portal_mapping}
+    exits = {portal.name for portal in portal_mapping}
```

Comment:
```suggestion
    entrances = {portal.name for portal in portal_mapping}
    exits = {portal.name for portal in portal_mapping}
    entrance.add("Shop Portal")
    exits.add("Shop Portal")
```

Shops aren't in the portal mapping list because they're weird